### DISCUSSION TITLE
Discussion Open! / 讨论区开放！

### DISCUSSION THREAD

**Discussion by sanmusen214** [BlueArchiveArisHelper/BAAH#80] [General]
讨论区现已开放，欢迎开贴讨论任何有关BAAH的使用问题或者探讨项目代码.
Feel free to write a post to talk about any questions when using BAAH or discuss about the project code.

**Comment by CMRG21** [comment]
1.8.9版本，国际服活动图设置关闭自动推剧情和关卡
只设置了第一天，其中有三项，关卡10*20，关卡11*20，关卡12*-1。前两项不启用，第三项启用。
执行时三者均执行。也就是说均被视为启用。
希望修复这个bug

**Reply by sanmusen214** [reply]
已修复，下次提issue
